### PR TITLE
Feature/01  mfg website

### PR DESCRIPTION
**Pull request by TaylorFoster90**
All 10 block Elements

### CODE REVIEW

**Comment by eklemen** [comment]
:+1:

**Comment by mfees** [comment]
Looks good mostly. I would just say look at the support section again on each, I think he wanted the browsers that the tag can be supported in.
Otherwise 
:+1:

**Comment by TaylorFoster90** [comment]
https://github.com/TheIronYard--Orlando/FEE--2015--SPRING/issues/23

**Comment by mfees** [comment]
:+1:

File: docs/HTML.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+* *content:* Flow Content
+* *support:* all browsers
+* *example:*
+``` <p> Some text </p>
+ <hr>
+<p> some other text </p> ```
+<p> some Text </p>
+<hr>
+<p> some other text </p>
+
```
Comment (al-the-x) [inline docs/HTML.md:22]: Missing code fence here... Whoops! Check the formatting with a visual "sanity check" on Github to avoid these mistakes...

Comment (al-the-x) [inline docs/HTML.md:22]: Oh, and another clue is how the syntax highlighting doesn't change colors past this point.

File: docs/HTML.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+<ul> <li> Thing One </li> <li> Thing Two </li> <li> Thing Three </li> </ul>
+
+### 6. `<ol>` -- Ordered List
+* *type:* block-level
+* *content:*  flow content/palpable content
+* *support:*  all browsers
+* *example:*
+```<ol> <li> One Thing </li> <li> Two Thing </li> <li> Three Thing </li> </ol>```
+<ol> <li> One Thing </li> <li> Two Thing </li> <li> Three Thing </li> </ol>
+
```
Comment (al-the-x) [inline docs/HTML.md:54]: This is a good place to document the `<li>` tag, since it's related to the previous two...

File: docs/HTML.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+<article> <h4> Heading </h4> <p> Articles should always include a heading </p> </article>
+
+### 8. `<dl>` -- Definition List
+* *type:* block-level 
+* *content:*  flow content/palpable content
+* *support:* all browsers
+* *example:*
+```<dl> <dt> Challenge </dt> <dd> A call to take part in contest or competition </dd> </dl> ```
+<dl><dt>Challenge</dt><dd>A call to take part in contest or competition </dd></dl>
+
```
Comment (al-the-x) [inline docs/HTML.md:78]: Nice to see you start with the `<dl>` tag... Many others did not. Like above, this would be a good place to document the related tags and share an example block between them, particularly since a `<dl>` by itself is pretty worthless.

File: pub/about.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
+ <!DOCTYPE html>
+<html>
+<head>
+	<title>The Iron Yard | About</title>
```
Comment (al-the-x) [inline pub/about.html:4]: More than expected but not excessive. Great work grabbing the page title!

File: pub/about.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+</head>
+<body>
+ <nav> locations academy accelerator events about jobs press blog  </nav>
```
Comment (al-the-x) [inline pub/about.html:7]: Good choice of container with `<nav>`... but aren't those _links_ on the website?

File: pub/about.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+ <div> 
+ 	<h1> About The Iron Yard </h1>
+ 	<hr>
+ 	<nav> Team Culture History Contact </nav>
```
Comment (al-the-x) [inline pub/about.html:11]: Same here as above...

File: pub/about.html [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+<h4> The Iron Yard is a solution at that juncture of supply, demand, business and a broken education system. </h4>
+<p> We are committed to training the highest-quality developers and startups in the world, both in skill and in character. Our mission is to find people who want to pursue the craft and life-long adventure of technology, teach them the tools of the trade, and then release them into the world with the drive and capability to make a difference. We believe that passionate, talented people who love what they do will create great families, jobs, companies and solutions to the world's problems.</p>
+<h4> Join Us. </h4>
+<p>We want to accomplish our goal alongside talented, passionate people who share our vision, love what they do, and want to have a whole lot of fun along the way. Check out our job openings and join the team. </p>
+<hr>
+<footer> <nav> locations academy accelerator events about jobs news blog </nav> </footer>
```
Comment (al-the-x) [inline pub/about.html:25]: Same comment here as above about missing links, but I wanted to highlight your choice of the `<footer>` tag to surround this. Contrast that with the `<div>` you used above... Is there another more appropriate tag?

File: pub/culture.html [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+<body>
+<nav> locations academy accelerator events about jobs press blog  </nav>
+<div> 
+ 	<h1> Our Culture </h1> 
+ </div>
+ <hr>
+ <p>If you walk into The Iron Yard on any given afternoon, you're as likely to see an engineering team architecting software as you are to see a group enjoying craft brews over ping pong and video games. We've been told we're a "work hard, play hard" crew, and we couldn't agree more. Our standards for producing awesome stuff are extremely high, but so is our capacity to truly enjoy life alongside the people we work with.
+
+Here are a few characteristics of how we work:
+</p>
```
Comment (al-the-x) [inline pub/culture.html:15]: Not sure this worked out exactly as expected... There are a couple of paragraphs -- and thus some `<p>` tags -- that should be in that section.

**Comment by al-the-x** [comment]
So, I get the impression that you typed a lot of that copy yourself. Correct assumption? Tell me a little about how this assignment went for you... Bright bulbs, dim bulbs, what did you love, what did you hate...? Be brutally honest.

**Comment by TaylorFoster90** [comment]
@al-the-x assignment went great after studying the box model and how to manage where things need to be. 
If i had to choose something I hate, I just cant figure out whats the "right" way of doing things. For example: I'm not sure if "floating" certain things is the most efficient thing to do rather than margin manipulating. Or if its more efficient to stay way from a flood of <div>'s everywhere in the HTML and just making it look clean. Also, I do a lot of in-line styling in my html. I just love have to ability to control just that ONE thing i need to change. I've been doing research and some sites say to stay away from inline styling. 
So basically just having trouble understanding how to do it the professional way.
